content/docs/concepts/staking/_index.md

```diff
@@ -230,15 +246,32 @@ See [Bonding](/concepts/staking/#bonding) above.
 
-Unbonding is triggered by a staker submitting an `unbond()` transaction. Unbonding can begin as soon as the unbond transaction request has been finalised. On processing the transaction the bonded stake token moves from `bonded` to the intermediate state of `unbonding`. The stake is still locked during the unbonding period. The unbonding request is captured and tracked in memory and the staking transition is applied at the end of the epoch in which the unbonding period expires: unbonding is applied, reducing the validator’s bonded stake amount and so applying the voting power change to the validator.
+Unbonding is triggered by a staker submitting an `unbond()` transaction. Unbonding can begin as soon as the unbond transaction request has been finalized. On processing the transaction the bonded stake token moves from `bonded` to the intermediate state of `unbonding`. The stake is still locked during the unbonding period. The unbonding request is captured and tracked in memory. The staking transition is applied in two steps:
+
+- _at the end of the epoch in which the unbonding request was issued_: the validator's total bonded stake (and consequently [voting power](/glossary/#voting-power) decreases by the unbonded amount when the unbonding is applied at the end of the epoch,
+
+- _on expiry of the unbonding period_: NTN for the unbonding stake amount are released to the delegator at the end of the unbonding period.
+
 
 {{< alert title="Example" >}}
-Alice sends an `unbond()` tx at time `T`, a block in an epoch. Liquid Newton is burned at `T` and unbonding begins in the next block, `T+1`. The unbonding request is tracked in memory for application at the end of the epoch in which the unbonding period falls. At this point, the validator's bonded stake is reduced and Newton is unlocked and redeemed to Alice. Actual unbonding is then executed at `T+1` + `unbondingPeriod` + remainder of the epoch in which the `unbondingPeriod` expires.
+Alice sends an `unbond()` tx at time `T`, a block in an epoch. The tx is processed at `T` and an `UnbondingRequest` object for the necessary voting power change is created. At `T+1` the [unbonding period](/glossary/#unbonding-period) begins.
+
+The unbonding request is tracked in memory for application and:
+
+- at the end of the epoch in which `T` was processed the validator's bonded stake amount and voting power is reduced:
+  - the designated amount of Liquid Newton amount is unlocked and burnt if the stake being unbonded is [delegated](/glossary/#delegated),
+  - the amount of stake to reduce the unbonding pool by and Alice's share of the unbonding pool is calculated,
+  - the amount of Newton bonded to the validator is reduced by the unbonding amount, consequently reducing the validator's voting power.
+
+- on expiry of the unbonding period (`T+1` + `unbonding period`) Newton redemption (i.e. 'release') occurs when at that block height:
```

Comment:
To be more exact it is at the first epoch that ends after  `T+1` + `unbonding period`

Reply:
Thanks @lorenzo-dev1 ! Edited here and elsewhere in commit 320685b950e4b178702e6c910f34903c11e101f5.